### DISCUSSION TITLE
How to set theme at runtime?

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#11893] [Q&A]
With `AvaloniaLocator.Current.GetService<FluentAvaloniaTheme>()` no longer available in RC1.1, how can I set `Application.Current.ActualThemeVariant` at runtime which is read-only?

**Comment by rabbitism** [comment]
set ReqestedThemeVariant

**Comment by mysteryx93** [comment]
Thanks!

And what's the new way of using the `FocusManager` ?

    FocusManager.Instance.Focus(next, NavigationMethod.Directional);

**Reply by maxkatz6** [reply]
https://github.com/AvaloniaUI/Avalonia/wiki/Avalonia-11-Porting-Guide#focus

**Comment by mysteryx93** [comment]
This is not changing the theme to Dark.

    Application.Current!.RequestedThemeVariant = new ThemeVariant("Dark", new ThemeVariant("Light", null))

**Reply by rabbitism** [reply]
Can you try to use ThemeVariant.Dark directly?

**Reply by timunie** [reply]
Yeah I think @rabbitism is right. Creating a new Theme-Variant will create an empty one. Better to call any existing one. 

```cs
Application.Current!.RequestedThemeVariant = ThemeVariant.Dark;
```
